### PR TITLE
fix: remove raw css file import from .res files in critical care recording

### PR DESCRIPTION
**Pull request by Pragati1610**
fixes: https://github.com/coronasafe/care_fe/issues/3292

### CODE REVIEW

**Comment by nihal467** [comment]
@khavinshankar can you check this

**Review by khavinshankar — APPROVED** [review]

**Comment by nihal467** [comment]
@Pragati1610 can you check the merge conflict

**Comment by khavinshankar** [comment]
@Pragati1610 can you please resolve the merge conflict and if that involves ui changes do use `need testing` label

**Comment by nihal467** [comment]
@Pragati1610 work on the merge conflict

**Comment by Pragati1610** [comment]
Thank you @rithviknishad

**Comment by khavinshankar** [comment]
@Pragati1610 @rithviknishad the deploy seems to be failing
image

File: src/style/index.css [hunk 1]
```diff
@@ -428,458 +416,466 @@ button:disabled,
...
+    padding-bottom: 0.5rem;
+    padding-left: 1rem;
+    padding-right: 1rem;
+    font-size: 0.875rem;
+    line-height: 1.25rem;
+    font-weight: 600;
+    border-width: 0;
+    background-color: #def7ec;
+    color: #046c4e;
+
```
Comment (rithviknishad) [inline src/style/index.css:898]:
```suggestion
}
```

**Comment by rithviknishad** [comment]
@khavinshankar could you check now?

**Review by khavinshankar — APPROVED** [review]

File: src/Components/CriticalCareRecording/PressureSore/CriticalCare__PressureSoreEditor.res [hunk 2]
```diff
@@ -5,7 +5,6 @@
 @val external innerHeight: int = "window.innerHeight"
 
 @val external document: 'a = "document"
-%%raw("import './styles.css'")
```
Comment (khavinshankar) [inline src/Components/CriticalCareRecording/PressureSore/CriticalCare__PressureSoreEditor.res:8]: @Pragati1610 @rithviknishad 
removing this is causing styling issues like when there is no highlighting based on push score

**Comment by khavinshankar** [comment]
@Pragati1610 what's the status of this one

File: src/Components/CriticalCareRecording/PressureSore/CriticalCare__PressureSoreEditor.res (hunk 2 above)
Comment (Pragati1610) [reply src/Components/CriticalCareRecording/PressureSore/CriticalCare__PressureSoreEditor.res:8]: Figuring out why that is happening when the styles in that file is just applied on a checkbox and it has been accounted for in the global css stylesheet

**Comment by nihal467** [comment]
@khavinshankar can you follow up on this PR

**Comment by Pragati1610** [comment]
resolving the conflicts

**Comment by nihal467** [comment]
@Pragati1610 what is the status on the PR

**Comment by nihal467** [comment]
@Pragati1610 clear the changes and try to get it merged this week itself 

cc: @khavinshankar

File: src/Components/CriticalCareRecording/PressureSore/CriticalCare__PressureSoreEditor.res (hunk 2 above)
Comment (Pragati1610) [reply src/Components/CriticalCareRecording/PressureSore/CriticalCare__PressureSoreEditor.res:8]: can you confirm if it is still happening @khavinshankar

**Comment by khavinshankar** [comment]
putting it on hold as this is just a refactor

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**